utils/dispatcher.py

```
@@ -57,119 +55,163 @@ def check(self, message: discord.Message) -> bool:
+
+    BACKGROUND: int = 100
+    """For non-urgent background processing."""
+
+
+# ---------------------------------------------------------------------------
+# Handler record
+# ---------------------------------------------------------------------------
+
+
+@dataclass
+class MessageFilters:
```

Flake8 (7.3.0)

[error] redefinition of unused 'MessageFilters' from line 30

(F811)